### PR TITLE
Fix Run Coding Algorithms button

### PR DESCRIPTION
**Pull request by haroutboujakjian**
This PR tries to fix the "Run Coding Algorithms" button on the Home page of the app. The following updates are made already

- Fix `CELERY_BROKER_URL` inside of docker network environment to reference container name instead of localhost
- Adjust celery start command to work with current version of celery
- Change `run_coding_algorithms` to batch process VA cause codes since it's possible to send too much data over the wire. This causes the celery worker to have a memory leak.

### CODE REVIEW

File: va_explorer/va_data_management/utils/coding.py [hunk 1]
```diff
@@ -105,15 +106,31 @@ def run_coding_algorithms():
...
-    causes, issues = run_interva5(verbal_autopsies_without_causes)
+    causes_list = []
+    issues_list = []
+    verbal_autopsies_without_causes_list = []
 
-    return verbal_autopsies_without_causes, causes, issues
+    # Determine number of iterations necessary for large lists of verbal autopsies
+    count = VerbalAutopsy.objects.filter(causes__isnull=True).count()
+    batches = ceil(count / 5)
+    batch_size = 500
```
Comment (ajbarnes) [inline va_explorer/va_data_management/utils/coding.py:119]:
```suggestion
    batch_size = 500
    batches = ceil(count / batch_size)
```

Comment (ajbarnes) [reply va_explorer/va_data_management/utils/coding.py:119]: This suggestion fixes a bug where `count / 5` leads to too many batches or runaway batch_start:batch_end in the loop.

File: va_explorer/va_data_management/utils/coding.py [hunk 2]
```diff
@@ -105,15 +106,31 @@ def run_coding_algorithms():
...
+        verbal_autopsies_without_causes = list(
+            VerbalAutopsy.objects.filter(causes__isnull=True)[batch_start:batch_end]
+        )
+        causes, issues = run_interva5(verbal_autopsies_without_causes)
+
+        causes_list += causes
+        issues_list += issues
+        verbal_autopsies_without_causes_list += verbal_autopsies_without_causes
+
+    return {"verbal_autopsies": verbal_autopsies_without_causes_list, "causes": causes_list, "issues": issues_list}
```
Comment (ajbarnes) [inline va_explorer/va_data_management/utils/coding.py:133]:
```suggestion
    return {
        "verbal_autopsies": verbal_autopsies_without_causes_list,
        "causes": causes_list,
        "issues": issues_list,
    }
```
Will fix the linter CI check

**Review by ajbarnes — CHANGES_REQUESTED** [review]
Pulled locally and almost everything works great. I left some suggestions to fix a small bug I discovered plus satisfy the lint CI check. Fix those and I say LGTM

**Review by haroutboujakjian — COMMENTED** [review]
Github is asking me to review these changes, but I submitted them so that doesn't make sense. In any case I'll approve?

**Review by ajbarnes — APPROVED** [review]

**Pull request merged**